### PR TITLE
[W5.11][F09-A1] Nicholas Ang

### PR DESCRIPTION
**Pull request by nicholasangcx**
2 enhancements on addressbook-level 2 with relevant documents updated for the 2kLoC learning outcome.

### CODE REVIEW

File: src/seedu/addressbook/commands/ListCommand.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
     public static final String COMMAND_WORD = "list";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Displays all persons in the address book as a list with index numbers.\n"
+            + ": Displays all persons in the address book in lexicographical order of name as a list with index numbers.\n"
```
Comment (RonakLakhotia) [inline src/seedu/addressbook/commands/ListCommand.java:16]: Ideally you should create a separate command like sortedList

File: src/seedu/addressbook/data/person/Name.java [hunk 2]
```diff
@@ -37,10 +38,15 @@ public static boolean isValidName(String test) {
     }
 
     /**
-     * Retrieves a listing of every word in the name, in order.
+     * Retrieves a listing of every word in the name, in lower case, in order.
      */
     public List<String> getWordsInName() {
-        return Arrays.asList(fullName.split("\\s+"));
+        List<String> wordsInName = Arrays.asList(fullName.split("\\s+"));
```
Comment (RonakLakhotia) [inline src/seedu/addressbook/data/person/Name.java:44]: \\s+ should be stored in a variable with a meaningful name and then used in the code.

File: src/seedu/addressbook/data/person/Person.java [hunk 3]
```diff
@@ -84,4 +84,9 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    @Override
+    public int compareTo(Person other) {
```
Comment (RonakLakhotia) [inline src/seedu/addressbook/data/person/Person.java:88]: Use better variable names, 'other' is not a good name. Also compareTo is not a descriptive name

File: src/seedu/addressbook/parser/Parser.java [hunk 4]
```diff
@@ -246,6 +246,10 @@ private Command prepareFind(String args) {
 
         // keywords delimited by whitespace
         final String[] keywords = matcher.group("keywords").split("\\s+");
+        for (int i=0; i<keywords.length; i++) {
```
Comment (RonakLakhotia) [inline src/seedu/addressbook/parser/Parser.java:249]: Coding standard violations in the for statement.

for (int i = 0; i < keywords.length; i++) {

**Review by RonakLakhotia — COMMENTED** [review]
Good attempt. Keep in mind of the coding standard violations.

**Comment by nicholasangcx** [comment]
closing

**Pull request closed without merging**